bin/observe

```diff
@@ -14,7 +14,7 @@ else
 fi
 
 tmux rename-window -t $session:$window 'observer'
-tmux send-keys -t $session:$window "watch docker-compose -f $basedir/../docker-compose.yml ps" C-m
+tmux send-keys -t $session:$window "watch 'docker-compose -f \'$basedir/docker-compose.yml\' ps'" C-m
```

Comment:
Are the use of single quotes valid here? 

> A single quote may not occur between single quotes, even when preceded by a backslash. 

https://www.gnu.org/software/bash/manual/html_node/Single-Quotes.html

(Disclaimer: I haven't tested this.)

Reply:
Since it’s wrapped in double quotes it’s valid. If you were to execute `watch 'docker-compose -f \'$basedir/docker-compose.yml\' ps'` bare that wouldn’t work (for two reasons); you’d need `watch "docker-compose -f '$basedir/docker-compose.yml' ps"`.

Reply:
I could rewrite this line as:
```bash
tmux send-keys -t $session:$window  "watch \"docker-compose -f '$basedir/docker-compose.yml' ps\"" C-m
```

But it isn’t strictly necessary.

Reply:
I like the symmetry of that version, so pushed the change

Reply:
Thanks for the explanation. I haven't had enough practice to wrap my head around bash's string substitution / interpolation / escape rules.